app/views/clients/_demographics.haml

```diff
@@ -6,23 +7,24 @@
       .row.c-card__content-blocks
         .col-md-12.col-lg-6.c-card__content-block
           %h5 Age
-          %span.c-card__content-block-text= @client.age
+          %span.c-card__content-block-text= @client.age || not_specified
           %span.c-card__content-block-text.col-md-12.col-lg-6.c-card__content-block-text--secondary= "(#{date_format(@client.date_of_birth)})" if @client.date_of_birth
         .col-md-12.col-lg-6.c-card__content-block
           %h5 Gender
-          %span.c-card__content-block-text= @client.gender.try(:text)
+          %span.c-card__content-block-text= find_and_preserve(@client.gender_descriptions.join(',<br>')).presence || not_specified
+
         .col-md-12.col-lg-6.c-card__content-block
           %h5 Race
-          %span.c-card__content-block-text= @client.race.try(:text)
+          %span.c-card__content-block-text= find_and_preserve(@client.race_descriptions.join(',<br>')).presence || not_specified
```

Comment:
Looks like this. I also added "not specified" to all fields.

<img width="442" alt="Screen Shot 2022-10-05 at 2 35 45 PM" src="https://user-images.githubusercontent.com/5333982/194137039-45354c30-2917-47e8-97d7-6240b3368fa3.png">

<img width="432" alt="Screen Shot 2022-10-05 at 2 40 08 PM" src="https://user-images.githubusercontent.com/5333982/194137167-434620cb-6821-4d7f-a77a-b922aca90471.png">

Reply:
This looks good.  Thanks